### PR TITLE
Add view in room to action bar in thread list

### PR DESCRIPTION
**Pull request by gsouquet**
Fixes https://github.com/vector-im/element-web/issues/20348

Screen Shot 2022-01-12 at 11 54 17

---
This PR currently has no changelog labels, so will not be included in changelogs.

Add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is plus `X-Breaking-Change` if it's a breaking change.

Preview: https://61df114f28128a07ce35dc8c--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/views/rooms/EventTile.tsx [hunk 1]
```diff
@@ -1446,10 +1457,10 @@ export default class EventTile extends React.Component<IProps, IState> {
                         </div>
                         <Toolbar className="mx_MessageActionBar" aria-label={_t("Message Actions")} aria-live="off">
                             <RovingAccessibleTooltipButton
-                                className="mx_MessageActionBar_maskButton mx_MessageActionBar_threadButton"
-                                title={_t("Reply in thread")}
-                                onClick={() => showThread({ rootEvent: this.props.mxEvent, push: true })}
-                                key="thread"
+                                className="mx_MessageActionBar_maskButton mx_MessageActionBar_viewInRoom"
+                                onClick={(e) => this.viewInRoom(e)}
```
Comment (t3chguy) [inline src/components/views/rooms/EventTile.tsx:1461]: can we make viewInRoom an arrow func and avoid this one?

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

**Review by janogarcia — CHANGES_REQUESTED** [review]
@gsouquet Thanks Germain for making progress with this! I haven't run the test build, but if it looks as in the screenshot, I’d point out that the [original proposal](https://github.com/vector-im/element-web/issues/20348) was to get rid of the overflow menu in this context, moving both actions — "View in room", and "Copy link to thread" — out of it.

**Comment by gsouquet** [comment]
Yeah apologies, I got mixed up, this is now fixed

Screen Shot 2022-01-12 at 17 23 02

**Review by janogarcia — APPROVED** [review]
@gsouquet Thanks for updating it! 👍

**Pull request merged**